auto_round/script/mllm.py

```diff
@@ -310,14 +326,15 @@ def tune(args):
     if args.quant_lm_head and args.low_gpu_mem_usage:
         print(f"warning, low_gpu_mem_usage=False is strongly recommended if the whole model could be loaded to "
               f"gpu")
-
-    autoround = round(model, tokenizer, dataset=args.dataset, extra_data_dir=args.extra_data_dir,
-                      bits=args.bits, group_size=args.group_size, sym=not args.asym,
-                      batch_size=args.batch_size, seqlen=seqlen, nblocks=args.nblocks, iters=args.iters,
-                      lr=args.lr, minmax_lr=args.minmax_lr, enable_quanted_input=not args.disable_quanted_input,
-                      amp=not args.disable_amp, nsamples=args.nsamples, low_gpu_mem_usage=args.low_gpu_mem_usage,
+    
+    autoround = round(model, tokenizer, image_processor=image_processor, dataset=args.dataset, 
+                      extra_data_dir=args.extra_data_dir, bits=args.bits, group_size=args.group_size,
+                      sym=not args.asym, batch_size=args.batch_size, seqlen=seqlen, nblocks=args.nblocks, 
+                      iters=args.iters, lr=args.lr, minmax_lr=args.minmax_lr, amp=not args.disable_amp,
+                      enable_quanted_input=not args.disable_quanted_input,
+                      nsamples=args.nsamples, low_gpu_mem_usage=args.low_gpu_mem_usage,
                       device=device_str, seed=args.seed, gradient_accumulate_steps=args.gradient_accumulate_steps,
-                      scale_dtype=args.scale_dtype, layer_config=layer_config,
+                      scale_dtype=args.scale_dtype, layer_config=layer_config, template=args.template,
```

Comment:
rename template to prompt_template to make it more easier to understand?

Reply:
This template is more like "model series template". It include both processor and special token.